notebooks/239-image-bind/239-image-bind-quantize-nncf.ipynb

```diff
@@ -0,0 +1,1570 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ov_quantized_modality_models = {
</pre><p>it is recommended to create core object once, then use the same object for all models compilation or if you do not need core at all, please use function instead:</p><p>from openvino.runtime import compile_model </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>